### DISCUSSION TITLE
Change cursor property to not-allowed when hovering uneditable items

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11264] [Other feature requests]
## The request

Small minor polish thing I just thought of. By setting the cursor CSS property to `cursor: not-allowed;` we could make it immediately clear to the user when they can't edit something. 

There's already a tooltip but the fade-in animation is rather slow imo, and having a bit more visual indication can't hurt.

![image](https://i.imgur.com/RNAaIDf.png)

**Comment by ajaffer91** [comment]
Hi @SeanPM5, i'm assuming you just want this to happen when hovering over the edit column and not the entire row?
